File: app/src/main/java/com/ericho/coupleshare/mvp/data/local/MyDbUpgradeListener.java
package com.ericho.coupleshare.mvp.data.local;

import org.xutils.DbManager;

/**
 * Created by steve_000 on 12/6/2017.
 * for project CoupleShare
 * package name com.ericho.coupleshare.mvp.data.local
 */

public class MyDbUpgradeListener implements DbManager.DbUpgradeListener {
    @Override
    public void onUpgrade(DbManager db, int oldVersion, int newVersion) {

    }
}

File: app/src/main/java/com/ericho/coupleshare/mvp/data/local/MyDbUpgradeListener.kt
package com.ericho.coupleshare.mvp.data.local

import org.xutils.DbManager

/**
 * Created by steve_000 on 8/7/2017.
 * for project CoupleShare
 * package name com.ericho.coupleshare.mvp.data.local
 */
class MyDbUpgradeListener: DbManager.DbUpgradeListener{
    override fun onUpgrade(db: DbManager?, oldVersion: Int, newVersion: Int) {

    }

}